js/matchkeys/goldrush/goldrush.js

```diff
@@ -0,0 +1,245 @@
+// Generate GoldRush match key.
+// Reads MARC JSON from input string.
+// Extracts and normalizes data from relevant fields.
+// Returns matchkey as string.
+//
+// Version: 1.0.0 (for specification September 2021)
+
+function loadMarcJson(marcJson) {
+  let marcObj;
+  try {
+    marcObj = JSON.parse(marcJson);
+  } catch (e) {
+    throw new Error(e.message);
```

Comment:
Unless you're adding some additional information to the error, this seems redundant and you should just let the "parse" fail and the exception will propagate.

You can add some context like this: `throw new Error('Input must be JSON', { cause: e });`

Reply:
Done.